src/System.Management.Automation/engine/BytePipe.cs

```diff
@@ -104,9 +104,7 @@ internal static FileBytePipe Create(string fileName, bool append)
             throw new RuntimeException(null, e, errorRecord);
         }
 
-        ApplicationInsightsTelemetry.SendExperimentalUseData(
-            ExperimentalFeature.PSNativeCommandPreserveBytePipe,
-            "f");
+        ApplicationInsightsTelemetry.SendExperimentalUseData("PSNativeCommandPreserveBytePipe", "f");
```

Comment:
`PSNativeCommandPreserveBytePipe` is no longer an experimental feature, so this telemetry call should be removed.

Reply:
We want to retain telemetry to monitor usage